src/appengine/handlers/base_handler_flask.py

```diff
@@ -181,7 +181,7 @@ def _add_security_response_headers(self, response):
     response.headers['X-Frame-Options'] = 'deny'
     return response
 
-  def render(self, path, values=None, status=200):
+  def render(self, path, values=None, status=200, response=None):
```

Comment:
Could you explain a bit why we need to add the response argument? Seems like we can always just add stuff to the response after the call to render() to keep the interface clean.

Reply:
yes, sorry. we actually don't need it. I will clean this approach.